docs/advanced-usage.md

```diff
@@ -53,6 +53,18 @@ steps:
 - run: java -cp java HelloWorldApp
 ```
 
+### Liberica
+```yaml
+steps:
+- uses: actions/checkout@v2
+- uses: actions/setup-java@v2
+  with:
+    distribution: 'liberica'
+    java-version: '11'
+    java-package: jdk # optional (jdk, jre, jdk+fx or jre+fx) - defaults to jdk
+- run: java -cp java HelloWorldApp
```

Comment:
Thank you for adding this example! Could you please add a link to it in the [README.md](https://github.com/actions/setup-java#advanced)

Reply:
sorry, I missed this point. I'll fix it now.